### PR TITLE
Added username in args of bonnie test

### PR DESCRIPTION
**Pull request by santhosh93**
Bonnie test expects the username in args.
So modified the args in the test.

Signed-off-by: Santhosh G <santhog4@linux.vnet.ibm.com>

### CODE REVIEW

File: perf/bonnie.py [hunk 1]
```diff
@@ -46,7 +47,8 @@ def setUp(self):
         """
 
         self.scratch_dir = self.params.get('dir', default=self.srcdir)
-        self.uid_to_use = self.params.get('uid-to-use', default=None)
+        self.username = getpass.getuser()
+        self.uid_to_use = self.params.get('uid-to-use', default=self.username)
```
Comment (apahim) [inline perf/bonnie.py]: `self.username` is not being used anywhere outside `setUp()`. No need to create it as an instance variable. Actually, no need to create it at all:
```python
from getpass import getuser
...

        self.uid_to_use = self.params.get('uid-to-use', default=getuser())
```

File: perf/bonnie.py [hunk 2]
```diff
@@ -68,8 +70,7 @@ def test(self):
         args.append('-d %s' % self.scratch_dir)
         args.append('-n %s' % self.number_to_stat)
         args.append('-s %s' % self.data_size)
-        if self.uid_to_use:
-            args.append('-u %s' % self.uid_to_use)
+        args.append('-u %s' % self.uid_to_use)
```
Comment (apahim) [inline perf/bonnie.py]: Ok

**Comment by santhosh93** [comment]
@apahim : Removed the self.username variable...

**Comment by apahim** [comment]
Ok, thank you. Looks good.

**Pull request merged**